rep-0147.rst

```diff
@@ -0,0 +1,512 @@
+.. Publishers are expected to choose only one.
+
+Acceleration Interface
+,,,,,,,,,,,,,,,,,,,,,,
+
+
+The messge to send a velocity command is ``geometry_msgs/Twist`` [#twist]_.
+The datatype is not timestamped or externall referenced as it is not intended to be used in an open loop control system.
+
+* Datatype: ``geometry_msgs/Twist`` [#twist]_
+* Topic: ``command_acceleration``
+
+::
+
+    geometry_msgs/Vector3 linear
+      float64 x
+      float64 y
+      float64 z
+    geometry_msgs/Vector3 angular
+      float64 x
+      float64 y
+      float64 z
+
+
+
+The command is a body relative set of accelerations in linear and angular space.
+The reference position is the origin of the ``base_link`` as defined in [#REP105]_
+
+
+.. TODO remove after review
+.. The acceleartion interface is implemented using the established ``mav_msgs/RateThrust`` [#ratethrust]_ message.
+.. This message is very similar to the standard ``Twist`` message however it removes some of the ambiguities of the reference frame by explicitly referencing the body frame.
+..
+.. * Datatype: ``mav_msgs/RateThrust``  [#ratethrust]_
+.. * Topic: ``command_acceleration``
+..
+.. ::
+..
+..   Header header
+..
+..   # We use the coordinate frames with the following convention:
+..   #   x: forward
+..   #   y: left
+..   #   z: up
+..
+..   geometry_msgs/Vector3 angular_rates  # Roll-, pitch-, yaw-rate around body axes [rad/s]
+..   geometry_msgs/Vector3 thrust         # Thrust [N] expressed in the body frame.
+..                                        # For a fixed-wing, usually the x-component is used.
+..                                        # For a multi-rotor, usually the z-component is used.
+..                                        # Set all un-used components to 0.
+
+
+High Level Interfaces
+---------------------
+
+Odometry Interface
+,,,,,,,,,,,,,,,,,,
+
+To communicate odometry information the ``nav_msgs/Odometry`` message will be used. [#odometry]_
+This is the standard message used throughout ROS systems already, many modules already take advantage of it.
+However there are many use cases for aerial vehicles when the estimation of the acceleration is important.
+So an additional message will be defined which provides acceleration in addition to position and velocity already defined.
+This additional message will be published in parallel.
+For backwards compatibility all systems will publish both.
+
+* Datatype: ``nav_msgs/Odometry`` [#odometry]_
+* Topic: ``odom``
+
+::
+
+  # This represents an estimate of a position and velocity in free space.
+  # The pose in this message should be specified in the coordinate frame given by header.frame_id.
+  # The twist in this message should be specified in the coordinate frame given by the child_frame_id
+  Header header
+  string child_frame_id
+  geometry_msgs/PoseWithCovariance pose
+  geometry_msgs/TwistWithCovariance twist
+
+* Datatype: ``nav_msgs/OdometryWithAcceleration`` `[proposed]`
+* Topic: ``odometry``
```

Comment:
I've seen that some packages use this topic instead of `odom`. Isn't it better to change the proposed topic name to something entirely new to avoid confusion, i.e. `odom_wacc`, `odom_full`, `odom_pta`? (It is difficult to come up with a new topic name that beats the simplicity of that though)